projectional/src/main/java/jetbrains/jetpad/projectional/cell/mapping/CellProvider.java

```diff
@@ -148,7 +171,7 @@ private Object getSourceFromProjectionalSynchronizer(ProjectionalRoleSynchronize
 
   private List<Cell> doGetCells(Mapper<?, ? extends Cell> mapper, Object actualSource) {
     if (mapper.getSource() == actualSource) {
-      return Collections.<Cell>singletonList(mapper.getTarget());
+      return Collections.singletonList(mapper.getTarget());
```

Comment:
GWT needs an explicit type parameter here for successful compilation. Get it back, please.